README.md

```diff
@@ -31,13 +56,13 @@ L:   1 | P:  14 | L006 | Operators should be surrounded by a single space unless
 L:   1 | P:  27 | L001 | Unnecessary trailing whitespace
 ```
 
-You can also have a play using [SQLFluff online](https://online.sqlfluff.com/).
+You can also have a play using [**SQLFluff online**](https://online.sqlfluff.com/).
 
-For full CLI usage and rules reference, see the docs.
+For full [CLI usage](https://docs.sqlfluff.com/en/stable/cli.html) and [rules reference](https://docs.sqlfluff.com/en/stable/rules.html), see [the SQLFLuff docs](https://docs.sqlfluff.com/en/stable/).
```

Comment:
```suggestion
For full [CLI usage](https://docs.sqlfluff.com/en/stable/cli.html) and [rules reference](https://docs.sqlfluff.com/en/stable/rules.html), see [the SQLFLuff docs](https://docs.sqlfluff.com/en/stable).
```

Reply:
Same comment here. We use trailing slashes on the docs website.